failover-slots/ngrok.yml

```diff
@@ -1,5 +1,5 @@
 version: "2"
-authtoken: 2f8iH6AbSmwLDZhLT0w0taIENbC_3WrBBLaeXYVc8v6Yq17DQ
```

Comment:
Gnarf, this should not have been here. Just revoked this token, so no harm done.